.github/workflows/post-release.yml

```diff
@@ -28,11 +28,11 @@ jobs:
           current_version=$(cargo metadata --format-version 1 --no-deps | \
             jq --raw-output '.packages | .[] | select(.name == "bevy").version')
           # Sanity check: current version should be 0.X.Y
-          if ! grep -q '^0\.[0-9]\+\.[0-9]\+$' <<< "${current_version}"; then
-            echo "Invalid version (not in 0.X.Y format): ${current_version}"
+          if ! grep -q '^0\.[0-9]\+\.[0-9]\+-dev$' <<< "${current_version}"; then
+            echo "Invalid version (not in 0.X.Y-dev format): ${current_version}"
             exit 1
           fi
-          minor_version=$(sed 's/^0\.\([0-9]\+\).*/\1/' <<< "${current_version}")
+          minor_version=$(sed 's/^0\.\([0-9]*\)\..*/\1/' <<< "${current_version}")
```

Comment:
```suggestion
          minor_version=$(sed 's/^0\.\([0-9]+\)\..*/\1/' <<< "${current_version}")
```

Reply:
yeah... but it doesn't work. and we don't need to be defensive against our inputs there

Reply:
I guess, but why not accept the more correct regex to not confuse the next person that looks at it? No reason to keep the slightly more wrong one.

Reply:
because it doesn't work? I tried using `+` but the command didn't work in my shell

Reply:
Does sed expect it to be `\+`?

Reply:
`\+` needs escaping, `*` doesn't, see my comment above

it's some kind of backward compatibility issue: star was always there as modifier, but plus came later, so they didn't want to break existing scripts...

Reply:
```
$ echo 'xxxxx' | grep 'x*'
xxxxx
$ echo 'xxxxx' | grep 'x\+'
xxxxx
```

Reply:
@rlidwka thanks for the explanation. Sounds kinda cursed 😅